_targets.R

```diff
@@ -22,13 +25,17 @@ list(
 
   # TODO: PULL SITE DATA HERE
   tar_map(
-    values = tibble(state_abb = states),
+    values = tibble(state_abb = states) %>%
+      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
     tar_target(nwis_inventory,
                oldest_active_sites %>% filter(state_cd == state_abb)),
     tar_target(nwis_data,
-               get_site_data(nwis_inventory, state_abb, parameter))
-    # Insert step for tallying data here
-    # Insert step for plotting data here
+               get_site_data(nwis_inventory, state_abb, parameter)),
+    tar_target(tally,
+               tally_site_obs(nwis_data)),
+    tar_target(timeseries_png,
+               plot_site_data(state_plot_files, nwis_data, parameter)),
+    names = state_abb
```

Comment:
Not an issue, and doesn't affect how the code runs (and the instructions do say to add it as the final argument), but I like having this `names` vector up at the top after `values`, so that I can tell how the targets will be named from the get-go.